webpack.config.js

```diff
@@ -31,12 +31,19 @@ module.exports = (env, { mode }) => ({
 	resolve: {
 		extensions: [".js", ".jsx"],
 		mainFields: ["browser", "main", "module"],
+		fallback: {
+			fs: false,
+			process: require.resolve("process/browser"),
+		}
```

Comment:
We already have the `NodePolyfillPlugin` below, which we can leverage. [Ref](https://www.npmjs.com/package/node-polyfill-webpack-plugin)

```js
new NodePolyfillPlugin({
  additionalAliases: ['process'],
})
```